src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -11,15 +11,15 @@
         </section>
         <ul>
           <li>
-            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red">mdi-close</v-icon>
-            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <v-icon v-if="isApplicantPerson" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="!isApplicantPerson"  color="red">mdi-close</v-icon>
+            <v-icon v-else>mdi-circle-small</v-icon>
             <span>An individual</span>
           </li>
           <li>
-            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red">mdi-close</v-icon>
-            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <v-icon v-if="isApplicantOrg" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="!applicantOrgs" color="red">mdi-close</v-icon>
+            <v-icon v-else>mdi-circle-small</v-icon>
             <span>A business or a corporation</span>
```

Comment:
this is for point#1 - fix validations

Default Applicant:
![image](https://user-images.githubusercontent.com/116035339/225464247-52d70fe3-b0b4-40b4-b747-b5b8174e383f.png)


Remove Person and add another Person
![image](https://user-images.githubusercontent.com/116035339/225463828-03b43563-9ceb-42e7-87c3-5a19e1c35772.png)

Remove Person and add Org
![image](https://user-images.githubusercontent.com/116035339/225463878-95bed5dc-9157-43b5-b6f4-51b4775a577f.png)

Reply:
^^ Looks great! 😃